src/components/panel/panel.js

```diff
@@ -1767,6 +1773,12 @@ MdPanelPosition.absPosition = {
   LEFT: 'left'
 };
 
+/**
+ * Margin between the edges of a panel and the viewport.
+ * @type {number}
+ */
+MdPanelPosition.viewportMargin = 5;
```

Comment:
5 -> 8 (everything in Material design is based on 8s.)

https://material.google.com/layout/responsive-ui.html#responsive-ui-grid